### ISSUE TITLE
Staff gets warning e-mail about no shifts even if in no-shifts department

### ISSUE THREAD

**Issue by earl7399** [magfest/ubersystem#2045]
What I expect to happen:
If I'm in a no-shifts department, I should not be warned that I'll lose my staff privileges if I don't sign up for shifts.

What happens:
I'm in a regular shift-based department by default and also in a no-shifts dept where I'll be working for the duration of the event. I received an e-mail stating "You're receiving space in one of our volunteer hotel rooms, which has a requirement of NUMBER weighted volunteer hours, and you're currently only signed up for 0.0 hours."

**Comment by binary1230** [comment]
I think this is related to another issue like this involving showing no shifts available, will look when I have more time.  basically same deal though.

**Comment by kitsuta** [comment]
Ahh, I'll bet that the email is sent as long as you're in any shift-based departments, whereas you want it to send only if you're _only_ in shift-based departments. Makes sense.

**Comment by bds002** [comment]
Yes to clarify in this instance Bob was assigned to both Dorasi and Registration. 
If someone is in both, we would want them to sign up for shifts in the shift weighted department.
Ideally we are starting to move all departments to shifts. (choosen or assigned).
This Email should be changed to add because you are assigned to c.department and haven't signed up for shifts yet....
